release_notes/47-release-notes.adoc

```diff
@@ -429,4 +429,9 @@ This release of {product-title-short} addresses the following security vulnerabi
 * link:https://access.redhat.com/security/cve/cve-2025-30204[CVE-2025-30204] Excessive memory allocation during header parsing in `golang-jwt` package.
 * link:https://access.redhat.com/security/cve/cve-2024-57083[CVE-2024-57083] Denial of Service vulnerability in the `redoc` package.
 
+It also contains the following bug fix:
+
+//ROX-28699
+* Fixed an issue where verifying multi-signed images failed due to incorrect error handling.
```

Comment:
Should this not be moved to below line 414 with the other bug fix?

Reply:
I think they are listed separately for each release, and I think there is only 1 bug fix for 4.7.2 (line 414 was fixed in 4.7.1). LMK if that's not right.